llvm/test/CodeGen/AMDGPU/llc-pipeline-npm.ll

```diff
@@ -0,0 +1,278 @@
+; RUN: llc -enable-new-pm -disable-verify -mtriple=amdgcn--amdhsa -print-pipeline-passes < %s 2>&1 \
+; RUN:   | FileCheck -check-prefix=GCN-O2 %s
+
+; RUN: llc -O3 -enable-new-pm -disable-verify -mtriple=amdgcn--amdhsa -print-pipeline-passes < %s 2>&1 \
```

Comment:
-disable-verify should be removed, it only disables the initial run of the IR verifier on the input